verilog/analysis/verilog_linter.cc

```diff
@@ -51,23 +50,6 @@
 #include "verilog/parser/verilog_token_classifications.h"
 #include "verilog/parser/verilog_token_enum.h"
 
-// TODO(hzeller): make --rules repeatable and cumulative
-ABSL_FLAG(verilog::RuleBundle, rules, {},
```

Comment:
I'd leave the flags here or in whatever compilation unit we define the constructor of LlinterConfigurationFlags (or, as I'd like to call it LinterConfigurationOptions). See comments below.

Reply:
Moved the flags back